lib/jsdom/level2/html.js

```diff
@@ -2102,6 +2121,16 @@ define('HTMLFrameElement', {
               self.removeEventListener('DOMNodeInsertedIntoDocument', loader, false);
               this._waitingOnInsert = false;
               loadFrame(self);
+
+              parent._length++;
+              refreshAccessors(self._ownerDocument);
+
+              var name = self.getAttribute("name");
```

Comment:
I'm confused why this is necessary here given the generic code above it, and why it doesn't just use `parent._frame` like the code above it does.